### PR TITLE
inital commit, base html structure

### PR DESCRIPTION
**Pull request by supernova-97**
created basic html structure

### CODE REVIEW

**Review by salaos — APPROVED** [review]
Looks good, nice code for the first step!

File: index.html [hunk 1]
```diff
@@ -7,6 +7,48 @@
     <title>Document</title>
 </head>
 <body>
-    
+  <header>
+    <nav id="nav-bar">
+       <a href="">That's me</a>
```
Comment (Gblur) [inline index.html:12]:
```suggestion
       <a href="#me">That's me</a>
```

File: index.html [hunk 2]
```diff
@@ -7,6 +7,48 @@
...
+       <a href="">Skills</a>
```
Comment (Gblur) [inline index.html:13]:
```suggestion
       <a href="#skills">Skills</a>
```

File: index.html [hunk 3]
```diff
@@ -7,6 +7,48 @@
...
+       <a href="">Projects & Websites</a>
```
Comment (Gblur) [inline index.html:14]:
```suggestion
       <a href="#projects">Projects & Websites</a>
```

File: index.html [hunk 4]
```diff
@@ -7,6 +7,48 @@
...
+       <a href="">CV</a>
```
Comment (Gblur) [inline index.html:15]:
```suggestion
       <a href="#cv">CV</a>
```

File: index.html [hunk 5]
```diff
@@ -7,6 +7,48 @@
...
+       <a href="">Contact</a>
```
Comment (Gblur) [inline index.html:16]:
```suggestion
       <a href="#contact">Contact</a> 
```

**Review by Gblur — CHANGES_REQUESTED** [review]

**Pull request merged**